### PR TITLE
Add recipe for Python package numpy-stl

### PR DESCRIPTION
**Pull request by csachs**
Current workaround with patch removing unnecessary `nine` dependency will be removed in the future (it's fixed in upstream [ WoLpH/numpy-stl#67 ]). @WoLpH agreed to be added as maintainer.

### CODE REVIEW

**Comment by csachs** [comment]
Not necessarily the type of error I expected …
```
The following specifications were found to be in conflict:
  - numpy 1.9* -> python 2.6* -> vc 9
  - python 3.6*
```
Just for one Windows build. Will try to fix this …

**Comment by csachs** [comment]
Excuse my inactivity on this PR for the past months, were quite busy.
Now I've fixed the problems and migrated the recipe style to the latest flavor.
Should be fine now?

**Comment by csachs** [comment]
I fear this PR might've drifted a bit far back in the list of open PRs, but I'd say it's ready as-is … pinging
@conda-forge/staged-recipes for review (I hope that's okay).
Thanks!

File: recipes/numpy-stl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - stl = stl.main:main
+    - stl2ascii = stl.main:to_ascii
+    - stl2bin = stl.main:to_binary
+
+requirements:
+  build:
+    - setuptools
```
Comment (bgruening) [inline recipes/numpy-stl/meta.yaml]: I think setuptools should be in the host section.

Comment (csachs) [reply recipes/numpy-stl/meta.yaml]: Okay, moved it to the `host` section. Sorry, my first conda-build 3 recipe!

**Pull request merged**